suspect/io/twix.py

```diff
@@ -382,44 +382,53 @@ def load_twix(filename):
 
 def anonymize_twix_header(header_string):
     """Removes the PHI from the supplied twix header and returns the sanitized version.
-
     This consists of:
     1) Replacing the patient id and name with strings of lower case x
     characters.
-    2) Replacing the patient birthday with 19700101
+    2) Replacing the patient birthday with 19900101
     3) Replacing the patient gender with the number 0
-    4) All references to the date and time of the exam have all alphanumeric
-    characters replaced by lower case x. Other characters (notably the period)
-    are left unchanged.
-
+    4) Replacing the patient age and weight with 00.000000
+    5) Replacing the patient height with 0000.000000
+  
     Parameters
     ----------
     header_string : str
         The header string to be anonymized
-
```

Comment:
Please can we keep the existing white space in docstrings, I think it adds to the readability and matches the numpy spec.